### PR TITLE
(SIMP-4009) Add OEL and Puppet 5 Support

### PR DESCRIPTION
**Pull request by trevor-vaughan**
SIMP-4009 #comment update postfix

### CODE REVIEW

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-postfix/pipelines/24856878

**Comment by jeefberkey** [comment]
CHANGELOG should be reviewed

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,7 @@
+* Tue Mar 13 2018 Trevor Vaughan <tvaughan@onyxpoint.com> - 5.1.1-0
+- Ensure that only IPv4 is used if IPv6 is disabled on the system
```
Comment (lnemsick-simp) [inline CHANGELOG]: changelog is out of time order and doesn't  properly reflect what has been released

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-postfix/pipelines/25872168

**Comment by ayohrling** [comment]
Acceptance tests updated for new nodeset style: https://gitlab.com/simp/pupmod-simp-postfix/pipelines/25879183

**Pull request merged**